### PR TITLE
Getting Opportunity Items in Request for Quotation Document by Clicking "Get Items From" button

### PR DESCRIPTION
**Pull request by AravindPranera**
In Request for Quotation document when we click Get items from button, it will show dropdown consists of Opportunity along with Material Request and Possible Supplier. By choosing Opportunity, it will display a dialog box of Opportunity List with Check box. We can select the Opportunity list and click on "Get Items" . We can get Opportunity list items in Request for Quotation Document.

### CODE REVIEW

**Comment by mbauskar** [comment]
@AravindPranera ,

Thanks for contributing!

Our contribution guidelines require that you add a screenshot showing that your proposed change works with an example also can you please check the travis and codecy build. This pull request will be closed, but you can re-open it after adding the requested screenshot.

https://github.com/frappe/erpnext/wiki/Contribution-Guidelines

**Comment by AravindPranera** [comment]
Here is a screenshots.

image
image
image

File: erpnext/buying/doctype/request_for_quotation/request_for_quotation.js [hunk 1]
```diff
@@ -44,100 +44,13 @@ frappe.ui.form.on("Request for Quotation",{
...
-						frm.reload_doc();
 					}
 				});
 			});
-		}
+		}		
 
 	},
 
-	get_suppliers_button: function (frm) {
```
Comment (rmehta) [inline erpnext/buying/doctype/request_for_quotation/request_for_quotation.js]: why remove this?

File: erpnext/buying/doctype/request_for_quotation/request_for_quotation.js [hunk 2]
```diff
@@ -254,6 +144,27 @@ erpnext.buying.RequestforQuotationController = erpnext.buying.BuyingController.e
 						}
 					})
 				}, __("Get items from"));
+
+			// Get items from Opportunity 
+            this.frm.add_custom_button(__('Opportunity'),
```
Comment (rmehta) [inline erpnext/buying/doctype/request_for_quotation/request_for_quotation.js]: whitespace!

File: erpnext/crm/doctype/opportunity/opportunity.py [hunk 3]
```diff
@@ -84,19 +84,11 @@ def on_trash(self):
 		self.delete_events()
 
 	def has_active_quotation(self):
-		if not self.with_items:
```
Comment (rmehta) [inline erpnext/crm/doctype/opportunity/opportunity.py]: why change this?

**Review by rmehta — CHANGES_REQUESTED** [review]

File: erpnext/buying/doctype/request_for_quotation/request_for_quotation.js (hunk 1 above)
Comment (AravindPranera) [reply erpnext/buying/doctype/request_for_quotation/request_for_quotation.js]: Sorry, Add this line.

File: erpnext/buying/doctype/request_for_quotation/request_for_quotation.js (hunk 2 above)
Comment (AravindPranera) [reply erpnext/buying/doctype/request_for_quotation/request_for_quotation.js]: Remove the whitespace.

File: erpnext/crm/doctype/opportunity/opportunity.py (hunk 3 above)
Comment (AravindPranera) [reply erpnext/crm/doctype/opportunity/opportunity.py]: Add this line too.

**Review by AravindPranera — COMMENTED** [review]
I have added comments for your review.

**Comment by saurabh6790** [comment]
@AravindPranera Travis and Codacy are failing! Please fix them and reopen the PR.

**Comment by rmehta** [comment]
@AravindPranera you still need to add back the functions you may have accidentally deleted. Can you fix this?

**Comment by AravindPranera** [comment]
​I have committed the files with all functions, kindly merge it.

Thanks,
Aravind.​

Pranera Services & Solutions.
Tiruppur.

**Comment by AravindPranera** [comment]
Thanks,
Aravind.

Pranera Services & Solutions.
Tiruppur.

**Comment by rmehta** [comment]
👍  for first PR 😄

**Pull request merged**